geometry_msgs/Point32[] points
int32 leftBound
int32 rightBound
float64 distanceToLeftCluster
float64 distanceToRightCluster
float64 slope
float64 y_intercept
